admin/create-dumps.sh

```diff
@@ -95,7 +95,7 @@ elif [ "$DUMP_TYPE" == "incremental" ]; then
 elif [ "$DUMP_TYPE" == "feedback" ]; then
     SUB_DIR="spark"
 else
-    echo "Dump type must be one of 'full', 'incremental' or 'feedback'"
+    echo "ERROR: Dump Type $DUMP_TYPE is invalid. Dump type must be one of 'full', 'incremental' or 'feedback'"
```

Comment:
Whenever I see this message in the logs, its not entirely obvious to me something failed. I have to come back to the script to check it. So print ERROR to make it obvious.